exo-create/helpers/helpers.go

```diff
@@ -0,0 +1,67 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+const projectJsonContent = `
+{
+  "AppName": "my-app",
+  "ExocomVersion": "0.22.1",
+  "AppVersion": "0.0.1",
+  "AppDescription": ""
+}
+`
+
+const applicationYmlContent = `name: {{AppName}}
```

Comment:
`project.json` is a tmp file and is only used by `boilr`. `boilr` can parse it fine so I think we can leave it as it is?